plugins/obs-filters/data/blend_sub_filter.effect

```diff
@@ -37,7 +38,7 @@ float4 PSSubtractImageRGBA(VertDataOut v_in) : TARGET
 	float4 rgba = image.Sample(textureSampler, v_in.uv) * color;
 
 	float4 targetRGB = target.Sample(textureSampler, v_in.uv2);
-	rgba.rgb = saturate(rgba.rgb - targetRGB.rgb);
+	rgba.rgb = lerp(rgba.rgb, saturate(rgba.rgb - targetRGB.rgb), opacity * 0.01);
```

Comment:
Scaling of opacity should be done on the CPU, not GPU.

Reply:
Xaymar, I am unclear what you mean here. Are you referring to the `opacity * 0.01` code or are you referring to the lerp operation itself?

Reply:
I am referring to the `* 0.01`, as there is no need to do this per fragment.